src/components/views/auth/CaptchaForm.tsx

```diff
@@ -126,19 +126,22 @@ export default class CaptchaForm extends React.Component<ICaptchaFormProps, ICap
         let error = null;
         if (this.state.errorText) {
             error = (
-                <div className="error">
+                <div className="error" role="alert">
                     { this.state.errorText }
                 </div>
             );
         }
 
         return (
-            <div ref={this.recaptchaContainer}>
-                <p>{ _t(
+            <div className="mx_recaptchaContainer" ref={this.recaptchaContainer}>
```

Comment:
We prefix our classes with the component they are styling - so the classes you've added should be `mx_CaptchaForm_recaptchaContainer`, `mx_CaptchaForm_recaptchaContainerInfo` etc
This is explained a bit more in the css rules in the developer guide https://github.com/matrix-org/matrix-react-sdk/blob/develop/README.md#developer-guide

Reply:
Thanks for letting me know about that guide.

Reply:
Addressed: https://github.com/matrix-org/matrix-react-sdk/pull/8208/commits/94254772e0b763b2a95aa39801df8f40460a7a36